db/migrate/20150818181427_update_tenant_divisible_on_existing_rows.rb

```diff
@@ -1,7 +1,8 @@
 class UpdateTenantDivisibleOnExistingRows < ActiveRecord::Migration
   class Tenant < ActiveRecord::Base
   end
```

Comment:
@Fryguy huh? At the time this migration runs, the table will never have a `type` column.